File: app_manager/appmanager/src/main/java/app/editors/manager/managers/exceptions/NoConnectivityException.java
package app.editors.manager.managers.exceptions;

import java.io.IOException;

public class NoConnectivityException extends IOException {

    public NoConnectivityException() {
        super();
    }

    public NoConnectivityException(String message) {
        super(message);
    }

    public NoConnectivityException(String message, Throwable cause) {
        super(message, cause);
    }

    public NoConnectivityException(Throwable cause) {
        super(cause);
    }

    @Override
    public String getMessage() {
        return "No connectivity exception";
    }

}

File: app_manager/core/src/main/java/app/documents/core/network/common/exceptions/NoConnectivityException.kt
package app.documents.core.network.common.exceptions

import java.io.IOException

class NoConnectivityException : IOException {
    constructor() : super()
    constructor(message: String?) : super(message)
    constructor(message: String?, cause: Throwable?) : super(message, cause)
    constructor(cause: Throwable?) : super(cause)

    override val message: String
        get() = "No connectivity exception"
}